ajax/libs/overlayscrollbars/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "overlayscrollbars",
+  "filename": "OverlayScrollbars.min.js",
```

Comment:
Should be `js/OverlayScrollbars.min.js` the path to `filename` is the relative path to `basePath`